cmd/azmount/filemanager/cache_test.go

```diff
@@ -207,7 +207,7 @@ func Test_GetBlock_TestBounds(t *testing.T) {
 	ClearCache()
 
 	err, _ := GetBlock(-1)
-	if err.Error() != "invalid block index (-1)" {
+	if err.Error() != "Invalid block index (-1)" {
```

Comment:
This requires refactoring of the `GetBlock` function, but it is much better to do error checking via a type than the contents of the error.  `BlockIndexError` and `if errors.Is(err, BlockIndexError)` is much more stable than the string checks.